mlflow/tracing/export/mlflow.py

```diff
@@ -37,48 +38,40 @@ def __init__(self, client: TraceClient):
         self._client = client
         self._trace_manager = InMemoryTraceManager.get_instance()
 
-    def export(self, spans: Sequence[MlflowSpanWrapper]):
+    def export(self, spans: Sequence[ReadableSpan]):
         """
         Export the spans to MLflow backend.
 
         Args:
-            spans: A sequence of MlflowSpanWrapper objects to be exported. The base
-                OpenTelemetry (OTel) exporter should take OTel spans but this exporter
-                takes the wrapper object, so we can carry additional MLflow-specific
-                information such as inputs and outputs.
+            spans: A sequence of OpenTelemetry ReadableSpan objects to be exported.
         """
-        for span in spans:
-            if not isinstance(span, MlflowSpanWrapper):
-                _logger.warning(
-                    "Span exporter expected MlflowSpanWrapper, but got "
-                    f"{type(span)}. Skipping the span."
-                )
-                continue
-
-            self._trace_manager.add_or_update_span(span)
```

Comment:
note: In current implementation, all spans are added to the in-memory trace manager when they are created, so we don't need to iterate here.